### PR TITLE
Removed drone light in favor of night vision

### LINKED ISSUE #33447: You can see a drone's light while they are in the vents!

### PR DESCRIPTION
**Pull request by Garen7**
Fixes #33447

This is because drone's light could interfere with shadowlings and some other cases, and also because there was a bug where drone's lights could be seen from outside of vents when they were vent crawling.

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: AverageJoe82
add: drones now have night vision
del: drones no longer have lights
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

**Comment by optimumtact** [comment]
you don't need optional name here, unless you want to specify aname other than your github account name.

Whatever you do, don't keep optional name here, otherwise you'll show up as that!

**Comment by AverageJoe82** [comment]
oops, I forgot to change that thanks!

File: code/modules/mob/living/simple_animal/friendly/drone/verbs.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	to_chat(src, "<b>Drone Laws</b>")
 	to_chat(src, laws)
 
+/* Removed in favor of night vision
```
Comment (ShizCalev) [inline code/modules/mob/living/simple_animal/friendly/drone/verbs.dm]: Just remove everything that's commented out, we can see it in the git history anyway.

**Review by ShizCalev — DISMISSED** [review]

File: code/modules/mob/living/simple_animal/friendly/drone/verbs.dm (hunk 1 above)
Comment (AverageJoe82) [reply code/modules/mob/living/simple_animal/friendly/drone/verbs.dm]: alright cool, I didn't know that

**Review by Cyberboss — APPROVED** [review]

**Comment by Bawhoppen** [comment]
People have some false idea in their heads that drones aren't supposed to have any influence whatsoever on rounds

File: code/modules/mob/living/simple_animal/friendly/drone/verbs.dm [hunk 2]
```diff
@@ -12,21 +12,6 @@
 	to_chat(src, "<b>Drone Laws</b>")
 	to_chat(src, laws)
 
-/mob/living/simple_animal/drone/verb/toggle_light()
-	set category = "Drone"
-	set name = "Toggle drone light"
-	if(stat == DEAD)
-		to_chat(src, "<span class='warning'>There's no light in your life... by that I mean you're dead.</span>")
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/drone/verbs.dm]: poor pun, you served us well.

Comment (Bombpants) [reply code/modules/mob/living/simple_animal/friendly/drone/verbs.dm]: I demand a new pun

Comment (AverageJoe82) [reply code/modules/mob/living/simple_animal/friendly/drone/verbs.dm]: [rest well my sweet prince](https://www.youtube.com/watch?v=bq54jb_5WCI)

**Pull request merged**